backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -810,6 +810,9 @@ def _create_ingress(self, id, path: str, port_number: int):
                 name=id,
             ),
             spec=client.V1IngressSpec(
+                ingress_class_name=None
+                if cfg["target"] == "local"
+                else "nginx",
```

Comment:
Instead deriving "nginx" from the target, which is not self-explaining, I'd directly pass the ingressClassName into the backend. In addition, it gives us the flexibility to use different ingressClassNames ("nginx" may have another name in different instances).